build/vanagon/components/ruby-2.6.rb

```diff
@@ -13,7 +13,7 @@
 
   pkg.environment 'optflags', '-O2'
 
-  special_flags = " --prefix=#{ruby_dir} --with-opt-dir=#{settings[:prefix]} --enable-dtrace --with-baseruby=#{host_ruby}"
```

Comment:
DTrace is old and, in the case of Debian, not included in the system at all. Not sure if it's really needed for this anyway.